examples/experimental/skip_test/weave_walkthrough.ipynb

```diff
@@ -0,0 +1,415 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5f8febdb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import weave\n",
+    "weave.use_frontend_devmode()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71ffc2ae",
+   "metadata": {},
+   "source": [
+    "## Intro\n",
+    "\n",
+    "Weave is new kind of visual development environment, designed for developing AI-powered software.\n",
+    "\n",
+    "Building and working with AI models is a fuzzy process. The work is iterative, experimental, analytical, and visual."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ba8961ce",
+   "metadata": {},
+   "source": [
+    "## Getting data into Weave"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0ce4e951",
+   "metadata": {},
+   "source": [
+    "Get started by saving an object to Weave. Here we save the integer 5."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2ec8fa33",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mynum = weave.save(5, 'mynum')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8eb4082e",
+   "metadata": {},
+   "source": [
+    "mynum is now a Weave Node"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6ac4911b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "type(mynum)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "524f10af",
+   "metadata": {},
+   "source": [
+    "A Weave Node represents a Weave Expression. Converting a Weave Node to a string shows the expression."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9450c0ac",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "str(mynum)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ac36edf6",
+   "metadata": {},
+   "source": [
+    "You can execute a Weave Node by calling weave.use on it"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "272038df",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "weave.use(mynum)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df1326f6",
+   "metadata": {},
+   "source": [
+    "Weave saves data as a Weave LocalArtifact by default. You can inspect local artifacts at `/tmp/weave/fs/admin/local-artifacts`.\n",
```

Comment:
I think it isn't very clear what "data" is - nodes or something else?

Reply:
thank you, great point! tried to rephrase like this: Weave saves all of its data—including integers, text, images, and any other user input, as well as any downstream objects produced by computation—as Weave LocalArtifacts by default.